tracker.rst

```diff
@@ -136,6 +136,23 @@ In the sidebar you can also find links to summaries for easy issues and
 issues with a patch.
 
 
+Disagreement With a Resolution on the Issue Tracker
+===================================================
+
+First, take some time to consider any comments made in association with the
+resolution of the tracker issue. On reflection, they may seem more reasonable
+than they first appeared.
+
+If you still feel the resolution is incorrect, then raise the question on
+`python-dev`_. If the consensus there supports the disputed resolution, please
+take any further objections to `python-ideas`_ (or some other forum). Further
```

Comment:
If python-dev has already said that the resolution was right, then sending them to python-ideas to continue to vent isn't productive. Also mention that people should not re-open an issue once a core developer has closed it.